src/Injector.php

```diff
@@ -18,59 +17,54 @@
 use Zend\Di\Resolver\InjectionInterface;
 use Zend\Di\Resolver\TypeInjection;
 
+use function array_pop;
+use function class_exists;
+use function implode;
 use function in_array;
+use function interface_exists;
+use function sprintf;
 
 /**
  * Dependency injector that can generate instances using class definitions and configured instance parameters
  */
 class Injector implements InjectorInterface
 {
-    /**
-     * @var Definition\DefinitionInterface
-     */
+    /** @var Definition\DefinitionInterface */
     protected $definition = null;
 
-    /**
-     * @var ContainerInterface
-     */
+    /** @var ContainerInterface */
     protected $container = null;
 
-    /**
-     * @var Resolver\DependencyResolverInterface
-     */
+    /** @var Resolver\DependencyResolverInterface */
     protected $resolver;
 
-    /**
-     * @var ConfigInterface
-     */
+    /** @var ConfigInterface */
     protected $config;
 
-    /**
-     * @var string[]
-     */
+    /** @var string[] */
     protected $instantiationStack = [];
 
     /**
      * Constructor
      *
-     * @param ConfigInterface|null $config A custom configuration to utilize. An empty configuration is used
-     *      when null is passed or the parameter is omitted.
-     * @param ContainerInterface|null $container The IoC container to retrieve dependency instances.
-     *      `Zend\Di\DefaultContainer` is used when null is passed or the parameter is omitted.
-     * @param Definition\DefinitionInterface $definition A custom definition instance for creating requested instances.
-     *      The runtime definition is used when null is passed or the parameter is omitted.
-     * @param Resolver\DependencyResolverInterface|null $resolver A custom resolver instance to resolve dependencies.
-     *      The default resolver is used when null is passed or the parameter is omitted
+     * @param ConfigInterface|null                      $config     A custom configuration to utilize. An empty configuration is used
```

Comment:
@webimpress another nitpick of the alignment rule's cs fixer. Now the max line length is exceeded.